resolvers/webpack/README.md

```diff
@@ -53,6 +53,16 @@ settings:
       config-index: 1   # take the config at index 1
 ```
 
+or with explicit config file absolute root path, relative to current project path:
+
+```yaml
+---
+settings:
+  import/resolver:
+    webpack:
+      config: '$dirname/webpack.dev.config.js'
```

Comment:
i'm confused about the choice of `$dirname` - first, `$PWD` like POSIX or `<rootDir>` like jest make sense, but also, node already has a convention here - `.`.

Reply:
ok, I'll go with `$PWD` I thought about it as well, thanks

Reply:
why not just a dot?